packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -93,6 +93,8 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
 
   private logger: any; // to avoid adding @ts-ignore everywhere
   private hasLoggedBrowserSerial: boolean;
+  public device: any;
```

Comment:
question: any reason you decided to make it public?

Reply:
Sorry. I must have been testing something. It shall remain private. Changed it